src/components/button/examples/button-click.tsx

```diff
@@ -1,18 +1,25 @@
 import { Component, h, State } from '@stencil/core';
 
 /**
- * With click handler
+ * With click handler, and success feedback
  *
  * The click handler in this example sets the attributes `loading` and
- * `disabled` to `true`. After 1 second, the `loading` attribute is set to
- * `false` again. After another 5 seconds, the button is once again enabled.
+ * `disabled` to `true`. After 2 seconds, the `loading` attribute is set to
+ * `false` again. After another 2 seconds, the button is once again enabled.
  *
  * When the `loading` attribute changes from `true` to `false`, the button
- * automatically displays a checkmark icon for 2 seconds. Note that our click
- * handler isn't actually involved in this.
+ * automatically displays a checkmark icon for 2 seconds, to visualize that the
+ * previously ongoing process just succeeded.
+ * Note that our click handler isn't actually involved in this.
+ *
+ * :::important
+ * The button should remain disabled as long as it is loading, to reflect that
+ * a process is ongoing and prevent users to retrigger the same process.
+ * As soon as the loading is completed, the button should be enabled again.
```

Comment:
@Kiarokh I think you've misunderstood the reason for the timeout that's setting the button to enabled again. But that's a good thing, because if you've misunderstood it, I'm sure that many of our consultants have too. So it needs clarification.

Funnily enough, I actually added som clarification regarding this to the new examples I made in my suggestion for a `limel-loading-button`, before I had seen this change of yours 😄

<img width="506" alt="image" src="https://user-images.githubusercontent.com/154725/160990377-c5defdf7-2799-479b-95d1-e94fa1015862.png">

But I had only added it as a comment in the example code, so someone only looking at the UI wouldn't see it. I'll add an explanation to the description of the example too.

Also, I've been thinking of perhaps adding an example with a small form, where the whole behaviour is a little more like it would be in reality. I think that might help.

Reply:
Awesome! 👏

Reply:
I really don't see why it should be possible to have a button that's both loading and clickable. What should happen if I click a loading button? Should it cancel the loading? 😄 

I think we can get rid of confusion (and those clarifications) by rendering
```jsx
<button disabled={loading || disabled} />
```

Reply:
you might be right @vicgeralds 🤔 I can't think about a scenario where a user needs to re-trigger or re-click the same button. I can only think that the button will re-trigger the same process if clicked. But would it? Should it?

Reply:
I'm breaking this whole discussion out by removing the commit updating the docs from this PR. Let's do that in a separate PR instead, to keep things as simple as we can.